### PR TITLE
DOC-3737 Update release alerts for Log4j 2.16.0 schedule

### PR DESCRIPTION
**Pull request by benpesterr3**

### CODE REVIEW

File: content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -18,25 +18,15 @@ weight: 1
 
 ## Corda Enterprise 4.6.6
 
-Corda Enterprise 4.6.6 is a patch release of Corda Enterprise that fixes an urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
-
 {{< warning >}}
-
-Upgrade now to avoid exposure to the [Apache Log4j 2 vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). This is the most secure way to mitigate any risks associated with this vulnerability.
-
+Patch 4.6.6 contains dependency Log4j 2.15.0. A new vulnerability has been discovered in version 2.15.0 of the log4j logging library, as described here: https://nvd.nist.gov/vuln/detail/CVE-2021-45046. Apache have now released version 2.16.0 of the library to address the issue. Corda Enterprise 4.8.5 is due for release December 17 2021.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md:22]:
```suggestion
Patch 4.6.6 contains dependency Log4j 2.15.0. A new vulnerability has been discovered in version 2.15.0 of the log4j logging library, as described here: https://nvd.nist.gov/vuln/detail/CVE-2021-45046. Apache has released version 2.16.0 of the library to address the issue. Corda Enterprise 4.8.5 is due for release December 17 2021.
```

File: content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -18,25 +18,15 @@ weight: 1
...
 {{< /warning >}}
 
-### Upgrade recommendation
-
-As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.8/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.8/enterprise/upgrading-index.md).
+Corda Enterprise 4.8.4 is a patch release of Corda Enterprise that attempted to fix an urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md:25]: Is the random '2' meant to be there?

File: content/en/platform/corda/4.6/open-source/release-notes.md [hunk 3]
```diff
@@ -19,14 +19,11 @@ title: Release notes
...
-Github: https://github.com/corda/corda.
+### Updated December 15 2021
 
-{{< warning >}}
-
-Upgrade now to avoid exposure to the [Apache Log4j 2 vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). This is the most secure way to mitigate any risks associated with this vulnerability.
-
-{{< /warning >}}
+{{< note >}}
+The Latest version of Corda 4.6 has been updated to Log4j 2.15.0. A further update is underway to address security issues with Log4j 2.15.0. Check the releases and alerts banner for more information and expected release of the new update.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.6/open-source/release-notes.md:25]:
```suggestion
The latest version of Corda 4.6 has been updated to Log4j 2.15.0. A further update is underway to address security issues with Log4j 2.15.0. Check the releases and alerts banner for more information and expected release date of the next update.
```

File: content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md [hunk 4]
```diff
@@ -19,25 +19,15 @@ weight: 1
 
 ## Corda Enterprise 4.8.4
 
-Corda Enterprise 4.8.4 is a patch release of Corda Enterprise that fixes an urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
-
 {{< warning >}}
-
-Upgrade now to avoid exposure to the [Apache Log4j 2 vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). This is the most secure way to mitigate any risks associated with this vulnerability.
-
+Patch 4.8.4 contains dependency Log4j 2.15.0. A new vulnerability has been discovered in version 2.15.0 of the log4j logging library, as described here: https://nvd.nist.gov/vuln/detail/CVE-2021-45046. Apache have now released version 2.16.0 of the library to address the issue. Corda Enterprise 4.8.5 is due for release December 17 2021.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md:23]:
```suggestion
Patch 4.8.4 contains dependency Log4j 2.15.0. A new vulnerability has been discovered in version 2.15.0 of the log4j logging library, as described here: https://nvd.nist.gov/vuln/detail/CVE-2021-45046. Apache has released version 2.16.0 of the library to address the issue. Corda Enterprise 4.8.5 is due for release December 17 2021.
```

File: content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md [hunk 5]
```diff
@@ -19,25 +19,15 @@ weight: 1
...
 {{< /warning >}}
 
-### Upgrade recommendation
-
-As a developer, you should **urgently upgrade** to the [latest released version of Corda](../../../../../en/platform/corda/4.8/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.8/enterprise/upgrading-index.md).
+Corda Enterprise 4.8.4 is a patch release of Corda Enterprise that attempted to fix an urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md:26]: There's that 2 again...

File: content/en/platform/corda/4.8/open-source/release-notes.md [hunk 6]
```diff
@@ -19,14 +19,12 @@ title: Release notes
...
 
-Download the latest version of Corda 4.8 now to fix the urgent security issue - CVE-2021-44228 - caused by the Apache Log4j 2 dependency.
-Github: https://github.com/corda/corda.
+### Updated December 15 2021
 
-{{< warning >}}
-
-Upgrade now to avoid exposure to the [Apache Log4j 2 vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228). This is the most secure way to mitigate any risks associated with this vulnerability.
+{{< note >}}
+The Latest version of Corda 4.8 has been updated to Log4j 2.15.0. A further update is underway to address security issues with Log4j 2.15.0. Check the releases and alerts banner for more information and expected release of the new update.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/open-source/release-notes.md:25]:
```suggestion
The latest version of Corda 4.8 has been updated to Log4j 2.15.0. A further update is underway to address security issues with Log4j 2.15.0. Check the releases and alerts banner for more information and expected release date of the next update.
```

File: content/en/release-notes/_index.md [hunk 7]
```diff
@@ -17,7 +17,15 @@ This page provides updates on all major alerts, fixes and forthcoming patch rele
 
 ## Apache Log4j announcement
 
-In response to news of the [Apache Log4j 2 vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), new patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM are in progress.
+### Updated December 15 2021
+
+{{< note >}}
+A new vulnerability has been discovered in version 2.15.0 of the log4j logging library, as described here: https://nvd.nist.gov/vuln/detail/CVE-2021-45046.
+Additional patches are in progress to all current supported software versions for this issue. New patches will upgrade Corda and CENM dependencies to Log4j 2.16.0.
```
Comment (michelemarsden) [inline content/en/release-notes/_index.md:24]:
```suggestion
Additional patches are in progress for all current supported software versions for this issue. New patches will upgrade Corda and CENM dependencies to Log4j 2.16.0.
```

File: content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md (hunk 2 above)
Comment (michelemarsden) [reply content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md:25]: And 'attempted to fix'. Did it not?

Comment (benpesterr3) [reply content/en/platform/corda/4.6/enterprise/release-notes-enterprise.md:25]: Yes, meant to be there, no it did not fix it. 4.8.5 will...

File: content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md (hunk 5 above)
Comment (benpesterr3) [reply content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md:26]: the 2 is fine

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**